src/components/LoadingBlocks.js

```diff
@@ -0,0 +1,47 @@
+import React from "react"
+import PropTypes from "prop-types"
+
+const LoadingBlocks = ({ numberOfBlocks = 3, animationDurationInSec = 1 }) => {
+  const renderStyleForBlock = (currentBlockNumber) => {
+    const duration = animationDurationInSec.toString() + "s"
+    return {
+      animation: `blocks_${numberOfBlocks}-shine_block${currentBlockNumber} ${duration} infinite ease-out`,
+    }
+  }
+
+  const renderBlocks = () => {
+    const blocks = []
+    for (let i = 1; i < numberOfBlocks + 1; i++) {
+      blocks.push(
+        <div
+          className={`block block${i}`}
+          key={`loading_block_${i}`}
+          style={renderStyleForBlock(i)}
+        />
+      )
+    }
+    return blocks
+  }
+
+  return (
+    <div
+      className={`loading-block-container blocks-animation${numberOfBlocks}`}
+    >
+      {renderBlocks()}
```

Comment:
Fixed in https://github.com/keep-network/website/pull/453/commits/fec842ad918cd53c1f8df7c6c22d0fe25d1ca494

Had to also add `.fill(0)` because `map` is not working on an array with `undefined` values (and such array is created when using constructor `Array(length)`)